app/schemas/report.py

```diff
@@ -113,6 +113,26 @@ class Fertilization(BaseModel):
     treatment_description: Optional[str]
 
 
+class Livestock(BaseModel):
```

Comment:
It was just for example pdf template, until we receive or agree on correct or test example of json-ld in general. It will be changed for correct data